cypress/integration/WidgetsForEnrollmentPages/WidgetsForEnrollmentDashboard.feature

```diff
@@ -1,90 +1,95 @@
 
-      Scenario: User can close the Enrollment Widget
-        Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
-        When you click the enrollment widget toggle open close button
-        Then the enrollment widget should be closed
+  Scenario: User can close the Enrollment Widget
+    Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
+    When you click the enrollment widget toggle open close button
+    Then the enrollment widget should be closed
 
-      Scenario: User can close and reopen the Enrollment Widget
-        Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
-        When you click the enrollment widget toggle open close button
-        And you click the enrollment widget toggle open close button
-        Then the enrollment widget should be opened
+  Scenario: User can close and reopen the Enrollment Widget
+    Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
+    When you click the enrollment widget toggle open close button
+    And you click the enrollment widget toggle open close button
+    Then the enrollment widget should be opened
 
-      Scenario: User can see the enrollment details
-        Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
-        Then the enrollment widget should be opened
-        And the user sees the enrollment status is Active
-        And the user sees the enrollment date
-        And the user sees the incident date
-        And the user sees the enrollment organisation unit
-        And the user sees the owner organisation unit
-        And the user sees the last update date
+  Scenario: User can see the enrollment details
+    Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
+    Then the enrollment widget should be opened
+    And the user sees the enrollment status is Active
+    And the user sees the enrollment date
+    And the user sees the incident date
+    And the user sees the enrollment organisation unit
+    And the user sees the owner organisation unit
+    And the user sees the last update date
 
-      # TODO DHIS2-11482 - The test cases related with enrollment status edit are flaky. Move them to unit tests. 
-      # Scenario: User can modify the enrollment from Active to Complete
-      #   Given you land on the enrollment dashboard page by having typed #/enrollment?enrollmentId=wBU0RAsYjKE
-      #   And the enrollment widget should be opened
-      #   And the user sees the enrollment status is Active
-      #   And the user opens the enrollment actions menu
-      #   When the user changes the enrollment status to complete
-      #   Then the user sees the enrollment status is Complete
+  # Scenario: User can modify the enrollment from Active to Complete
```

Comment:
Hey, 
I see you deleted the `# TODO DHIS2-11482 - The test cases related with enrollment status edit are flaky. Move them to unit tests` comment. Did you have a reason for that?
Thanks!

Reply:
no reason, it's probably due to merge conflict, will put it back